java/statestore/src/main/java/sleeper/statestore/transactionlog/DuplicateSnapshotException.java

```diff
@@ -15,7 +15,9 @@
  */
 package sleeper.statestore.transactionlog;
 
-public class DuplicateSnapshotException extends Exception {
+import sleeper.core.statestore.StateStoreException;
+
+public class DuplicateSnapshotException extends StateStoreException {
```

Comment:
A DuplicateSnapshotException doesn't happen during an interaction with the state store. Maybe it shouldn't be a StateStoreException?